app/js/actions/actionTypes.js

```diff
@@ -10,6 +10,7 @@ export default {
   SET_PATIENT: types('set_patient', BASIC_TYPES),
 };
 export const FETCH_LAB_ORDERS = 'FETCH_LAB_ORDERS';
+export const FETCH_LAB_ORDER_TRENDS = 'FETCH_LAB_ORDER_TRENDS';
```

Comment:
let's call this FETCH_LAB_TEST_RESULTS... it's not really orders you are fetching here, but results of a single test.